plugins/feeds/public/rulezskbruteforceblocker.py

```diff
@@ -0,0 +1,42 @@
+import logging
+from dateutil import parser
+from datetime import datetime, timedelta
+from core.observables import Ip
+from core.feed import Feed
+from core.config.config import yeti_config
+from core.errors import ObservableValidationError
+
+class RulezSKBruteforceBlocker(Feed):
+
+    default_values = {
+        "frequency": timedelta(hours=24),
+        "name": "RulezSKBruteforceBlocker",
+        "source": "http://danger.rulez.sk/projects/bruteforceblocker/blist.php",
+        "description": "This feed contains daily list of IPs from rules.sk",
+    }
+
+    def update(self):
+        since_last_run = datetime.now() - self.frequency
+        try:
+            r = self._make_request(headers={"User-Agent": "yeti-project"})
+            lines = r.content.splitlines()[1:-1]
+            for line in lines:
+                ip, date, count, id = filter(None, line.split("\t"))
+                ip_date = parser.parse(date.replace("# ", ""))
+                if ip_date > since_last_run:
+                    self.analyze(ip, ip_date, line)
+        except Exception as e:
```

Comment:
What kind of exception are you catching? Again, it's usually a good idea not to catch them here since that way they can be reflected in the web UI

Reply:
i don't remember but i removed that, i also think about to do sorted by date and only check/add new